packages/devextreme/js/__internal/grids/grid_core/editing/m_editing_form_based.ts

```diff
@@ -150,7 +151,7 @@ const editingControllerExtender = (Base: ModuleType<EditingController>) => class
     }
   }
 
-  _showEditPopup(rowIndex, repaintForm?) {
+  _showEditPopup(rowIndex, repaintForm?): void {
```

Comment:
Minor comment.

Let's try to add some types for the method's arguments
At first look it may be:
```suggestion
  _showEditPopup(rowIndex: number, repaintForm?: boolean): void {
```